### PR TITLE
Request to join fixes

### PR DESCRIPTION
**Pull request by robbiecarlton**

### CODE REVIEW

File: src/containers/community/CommunitySettings.js [hunk 1]
```diff
@@ -181,6 +187,13 @@ export default class CommunitySettings extends React.Component {
       }
     }
     this.setState({expand: {...expand, [section]: open || !expand[section]}})
+    if (goToJoinRequests) {
+      setTimeout(() => {
```
Comment (robbiecarlton) [inline src/containers/community/CommunitySettings.js:191]: curious if you can think of a better way of doing this. We have to wait for the section to expand before we can scrollTo the joinRequests, but I don't know how to do that using lifecycle methods, so I just did setTimeout.

**Comment by robbiecarlton** [comment]
@levity - good to merge?
(check my comment on CommunitySettings)

File: src/containers/community/CommunitySettings.js (hunk 1 above)
Comment (levity) [reply src/containers/community/CommunitySettings.js:191]: I think you can check for the change in componentDidUpdate. setTimeout is fine too if it works though

**Comment by levity** [comment]
:shipit:

**Pull request merged**